source/MRMesh/MRVector3.h

```diff
@@ -42,42 +42,36 @@ struct Vector3
     T lengthSq() const { return x * x + y * y + z * z; }
     T length() const { return std::sqrt( lengthSq() ); }
 
-    [[nodiscard]] Vector3 normalized() const
+    [[nodiscard]] Vector3 normalized() const requires std::floating_point<T>
     {
-        if constexpr ( std::is_floating_point_v<T> )
-        {
-            auto len = length();
-            if ( len <= 0 )
-                return {};
-            return ( 1 / len ) * (*this);
-        }
-        else
-        {
-            static_assert( dependent_false<T>, "normalized makes sense for floating point vectors only" );
-        }
```

Comment:
why delete static_assert?

Reply:
I have to add `requires` anyway. It makes `static_assert` unnecessary.

Reply:
ok